TimePicker/TimePickerBase.js

```diff
@@ -304,7 +304,7 @@ const TimePickerBase = kind({
 
 	computed: {
 		hasMeridiem: ({order}) => order.indexOf('a') >= 0,
-		meridiemPickerWidth: ({meridiem, meridiems}) => meridiems[meridiem].length * 2
+		meridiemPickerWidth: ({meridiem, meridiems}) => meridiems && meridiems[meridiem].length * 2
```

Comment:
```suggestion
		meridiemPickerWidth: ({meridiem, meridiems}) => meridiems?.[meridiem].length * 2
```

Reply:
Non must have but nice to have :)

Reply:
I thought about it but I wasn't sure it's working with `[meridiem]`.

Reply:
I confirmed it works.
Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#optional_chaining_with_expressions